spec/features/user_delete_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'deleting a user' do
+  context 'with admin privilege' do
+    it 'deletes user' do
+      create :user, name: 'Adam'
+      admin = create :user, name: 'Bob', admin: true
+      login_as admin
+      visit admin_users_url
+      click_on 'Remove', match: :first
+      expect(page).to have_text 'User has been removed.'
+    end
+  end
+  context 'with master of ceremonies privilege' do
+    it 'will not delete a user' do
+      create :user, name: 'Adam'
```

Comment:
We had a issue with selecting the correct person to remove. If we removed the admin, the page would redirect to the log in page. Since the roles page is sorted by name we decided to give names to the users we create in case there is a change in the factory with the defaults.